src/Tribe/Status/Template.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+/**
+ * Class Tribe__Tickets__Status__Template
+ *
+ * @since TBD
+ *
+ */
+abstract class Tribe__Tickets__Status__Template {
```

Comment:
"Template" carries a lot of baggage and using it here could cause confusion. How about `Tribe__Tickets__Status__Abstract`?